### PR TITLE
User role and search class translation added

### PR DESCRIPTION
**Pull request by ashish0910**
Before fix:

Screenshot 2019-03-17 at 9 44 47 AM

After fix:

Screenshot 2019-03-17 at 9 43 21 AM

Search Class translation also added .

### CODE REVIEW

File: dashboard/views/users.ejs [hunk 1]
```diff
@@ -110,7 +110,7 @@
                                               <% if(data.users[i].language =='pl') {%> <td>Polish</td> <%} %>
                                               <% if(data.users[i].language =='ibo') {%> <td>Igbo</td> <%} %>
                                               <% if(data.users[i].language =='yor') {%> <td>Yoruba</td> <%} %>
-                                              <td><%= data.users[i].role %></td>
+                                              <% if(query.role != 'admin') {%> <td data-l10n-id="student">Student</td><%} else {%> <td data-l10n-id="admin">Admin</td> <%} %>
```
Comment (tarunsinghal92) [inline dashboard/views/users.ejs]: use something like this instead
`<td data-l10n-id="{{query.role.toLowerCase()}}">Student</td>`

**Comment by ashish0910** [comment]
@tarunsinghal92 done the changes .

**Comment by llaske** [comment]
Do not work as expected on latest dev branch.
Could you see what happen?

**Comment by ashish0910** [comment]
@llaske done the fixes .

**Comment by llaske** [comment]
Nice. Thanks.

**Pull request merged**